src/redprl/categorical_judgment.sml

```diff
@@ -10,6 +10,8 @@ sig
   val MEM : 'a * 'a -> 'a redprl_jdg
   val TYPE : 'a -> 'a redprl_jdg
 
+  val fromAst : RedPrlAst.ast -> RedPrlAst.ast redprl_jdg
```

Comment:
I added `fromAst` so that `sortOfJudgment` is just a composition of `synthesis` and `fromAst` and this should prevent future oversights. I did not change the signature because I am not sure if this is a Good Idea.